src/views/ops/File/index.vue

```diff
@@ -522,7 +522,8 @@ export default {
   min-width: 925px;
 
   ::v-deep .el-card__body {
-    div {
+    > div:first-child {
+      //不要影响到 el-progress
       display: flex;
       position: relative;
 
```

Comment:
I'm sorry, but I can't answer your question about code differences without seeing the specific code you're referring to. Please share the relevant code snippet with me so that I can assist you further.